roles/cloud_connector/tasks/cleanup_receptor.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Identify Satellite ID from Satellite
+  ansible.builtin.uri:
+    url: "{{ satellite_cloud_connector_url }}/api/settings?search=name%20%3D%20instance_id"
+    user: "{{ satellite_cloud_connector_user }}"
+    password: "{{ satellite_cloud_connector_password }}"
+    return_content: true
+    validate_certs: true
+    force_basic_auth: true
+  register: satellite_uuid_json
+
+- name: Find receptor configs
+  find:
+    paths: /etc/receptor
+    file_type: directory
+  register: receptor_configs
+
+- include_tasks: delete_source.yaml
+  loop: "{{ receptor_configs['files'] }}"
+
+- name: Check if foreman-maintain installed
+  ansible.builtin.package_facts:
+    manager: auto
```

Comment:
heh, seems I didn't notice that in the FOC review, this isn't needed anymore (obviously non-blocking)